autosubmit/autosubmit.py

```diff
@@ -4209,7 +4223,33 @@ def rocrate(expid, path: Path):
 
         from autosubmit.provenance.rocrate import create_rocrate_archive
         return create_rocrate_archive(as_conf, rocrate_json, jobs, start_time, end_time, path)
+    
+    @staticmethod
+    def provenance(expid: str, rocrate: bool = False) -> None:
+        """
+        :param expid: experiment identifier
+        :type expid: str
+        :param rocrate: flag to enable RO-Crate
+        :type rocrate: bool
+        """
```

Comment:
You can also provide a short doc here for users & other developers. e.g.

```suggestion
        """
        Create the experiment provenance archive.

        :param expid: experiment identifier
        :type expid: str
        :param rocrate: flag to enable RO-Crate
        :type rocrate: bool
        """
```